db/schema.rb

```diff
@@ -121,11 +121,11 @@
     t.datetime "updated_at",                          null: false
     t.string   "role"
     t.string   "daac"
-    t.string   "provider"
-    t.string   "uid"
     t.integer  "failed_attempts",        default: 0,  null: false
     t.string   "unlock_token"
     t.datetime "locked_at"
+    t.string   "provider"
```

Comment:
did you make this change? I'm not sure how big a deal it is, but this file isn't supposed to be manually edited as it is auto-generated when migrations are run.

Reply:
No, what I did was had to rerun rake db:migrate on a brand new setup database and I think it rearranged the file.   I didn't touch it at least.